index.html

```diff
@@ -4265,9 +4314,32 @@ <h2 id="GainNode">
         <p>
           This node has no <a>tail-time</a> reference.
         </p>
-        <dl title=
-        "[Constructor(&lt;a&gt;BaseAudioContext&lt;/a&gt; context, optional &lt;a&gt;GainOptions&lt;/a&gt; options)]interface GainNode : &lt;a&gt;AudioNode&lt;/a&gt;"
-        class="idl">
+        <dl title="[Constructor(&lt;a&gt;BaseAudioContext&lt;/a&gt; context, optional &lt;a&gt;GainOptions&lt;/a&gt; options)]interface GainNode : &lt;a&gt;AudioNode&lt;/a&gt;" class="idl">
+          <dt>
+            Constructor
+          </dt>
+          <dd>
+          <p>
+          Let <var>gain</var> be a new <a>GainNode</a> object.
+          <a href=#audionode-constructor-common>Initialize the common part</a>
+          of <var>gain</var>, and return <var>gain</var>.
+          </p>
+          <dl class=parameters>
+            <dt>
+              BaseAudioContext context
+            </dt>
+            <dd>
+              The <a>BaseAudioContext</a> this new <a>GainNode</a> will be
+              <a href="#associated">associated</a> with.
+            </dd>
+            <dt>
+              optional GainOptions options
+            </dt>
+            <dd>
+              An optional initial gain value for this <a>GainNode</a>.
```

Comment:
|options| isn't the actual initial gain value. Not sure how best to describe this.  Maybe say nothing at all since there's a link to the description of the actual GainOptions dict?  Same goes for all of the following FooOptions dictionaries.